ui/component/commentCreate/view.jsx

```diff
@@ -12,7 +12,7 @@ import { useHistory } from 'react-router';
 import type { ElementRef } from 'react';
 import emoji from 'emoji-dictionary';
 
-const COMMENT_SLOW_MODE_SECONDS = 30;
+const COMMENT_SLOW_MODE_SECONDS = 0;
```

Comment:
We should still have *some* lower limit. maybe 3-5 seconds for a smaller stream right now just to be safe?